### PR TITLE
Upgrade to Chrome 59

### PR DESCRIPTION
**Pull request by alexeykuzmin**
/cc @tonyganch @kevinsawicki

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-58.0.3029.110
+59.0.3071.86
```
Comment (kevinsawicki) [inline VERSION]: 59.0.3071.104 just came out with several security fixes, we should upgrade to that version.

https://chromereleases.googleblog.com/2017/06/stable-channel-update-for-desktop_15.html

Comment (alexeykuzmin) [inline VERSION]: Makes sense. Ok.

Comment (alexeykuzmin) [inline VERSION]: Done in 8958d12cb8124163118f22e69b6c3b76faa834b6.

**Comment by zcbenz** [comment]
@alexeykuzmin I have sent you an invitation to join the maintainers group, once it is done can you create a branch in this repo and recreate the pull request? So your changes can run on our CI machines.

File: patches/blink_file_path.patch [hunk 2]
```diff
@@ -1,12 +0,0 @@
-diff --git a/third_party/WebKit/Source/core/fileapi/File.idl b/third_party/WebKit/Source/core/fileapi/File.idl
-index df954bc..4683b28 100644
---- a/third_party/WebKit/Source/core/fileapi/File.idl
-+++ b/third_party/WebKit/Source/core/fileapi/File.idl
-@@ -32,6 +32,7 @@
-     Exposed=(Window,Worker),
- ] interface File : Blob {
-     readonly attribute DOMString name;
-+    readonly attribute DOMString path;
```
Comment (zcbenz) [inline patches/blink_file_path.patch]: This patch is still needed, lots of apps (like Atom and VS Code) are relying on it.

File: patches/web_contents.patch [hunk 3]
```diff
@@ -15,30 +15,14 @@ index cf3733f..065df39 100644
...
    }
 +  }  // !view_
  
    if (browser_plugin_guest_ && !GuestMode::IsCrossProcessFrameGuest(this)) {
      view_.reset(new WebContentsViewGuest(this, browser_plugin_guest_.get(),
-@@ -4858,6 +4865,7 @@ void WebContentsImpl::RendererUnresponsive(
-   // We might have been waiting for both beforeunload and unload ACK.
-   // Check if tab is to be unloaded first.
-   if (rfhi->IsWaitingForUnloadACK()) {
-+#if 0
```
Comment (zcbenz) [inline patches/web_contents.patch]: This patch is still needed, it is used to suppress the `unresponsive` event when a `beforeunload` event handler spends too much time running (like showing a confirm dialog).

File: patches/blink_file_path.patch (hunk 2 above)
Comment (alexeykuzmin) [inline patches/blink_file_path.patch]: Restored

File: patches/web_contents.patch (hunk 3 above)
Comment (alexeykuzmin) [inline patches/web_contents.patch]: This exact patch is no longer applicable, the code around have been refactored.
I would suggest to continue the discussion in the new pr (#313).

**Comment by alexeykuzmin** [comment]
I have created a new pr #313 instead of this one. Closed.

**Pull request closed without merging**